theme_bistro/views/snippets/s_product_catalog.xml

```diff
@@ -28,16 +26,4 @@
         <div class="o_we_shape o_web_editor_Zigs_06"/>
     </xpath>
 </template>
-
-<template id="s_product_catalog_dish" inherit_id="website.s_product_catalog_dish">
-    <xpath expr="//li" position="inside">
-        <p class="s_product_catalog_dish_description border-top text-muted pt-1 o_default_snippet_text">
-            <em>Add the description of this dish here.</em>
-        </p>
-    </xpath>
-    <xpath expr="//*[hasclass('s_product_catalog_dish_name')]" position="attributes">
-        <attribute name="class" remove="s_product_catalog_dish_dot_leaders" separator=" "/>
-    </xpath>
-</template>
```

Comment:
Here again, let's avoid having to add a migration script... this theme could adjusts the `li`'s padding for example, wdyt?